src/app/src/test/kotlin/org/vaccineimpact/api/tests/app_start/AllowedOriginFilterTests.kt

```diff
@@ -0,0 +1,97 @@
+package org.vaccineimpact.api.tests.app_start
+
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.verifyZeroInteractions
+import org.junit.Test
+import org.vaccineimpact.api.app.app_start.AllowedOriginsFilter
+import org.vaccineimpact.api.test_helpers.MontaguTests
+import spark.Request
+import spark.Response
+import javax.servlet.http.HttpServletResponse
+
+class AllowedOriginFilterTests : MontaguTests()
```

Comment:
It would be good to see a test that tested something other than the admin portal? Maybe just mix it up across the existing tests.